integration-tests/src/test/kotlin/org/stellar/anchor/platform/test/Sep24End2EndTests.kt

```diff
@@ -49,7 +49,14 @@ class Sep24End2EndTest(
         socketTimeoutMillis = 300000
       }
     }
-  private val anchor = wallet.anchor(config.env["anchor.domain"]!!)
+  private val anchor =
+    wallet.anchor(config.env["anchor.domain"]!!) {
+      install(HttpTimeout) {
```

Comment:
FYI: we use okhttp iirc default timeout is 1 minute

Reply:
I am curious of the root cause of the time out when calling `http://localhost:8080/sep24/transactions/deposit/interactive`. The call should return shortly. What causes the timeout for 1 minute?

Reply:
I'll create a Jira to investigate the timeouts. The anchor client isn't timing out anymore but I'm still seeing some `SocketTimeoutExceptions` from ktor occasionally.